python/py-netifaces/Portfile

```diff
@@ -14,18 +14,17 @@ description         an easy way to get the address(es) of the machine’s networ
 long_description    ${description}
 
 homepage            https://alastairs-place.net/projects/netifaces/
-master_sites        ${homepage}
-
 distname            netifaces-${version}
 
-checksums           rmd160  e5204d8415d1ae1ec84118eeb86928a1ea770f18 \
-                    sha256  53a711038db3d6e5cf7129d6f751185fd052f830f12f271fbfe17bd2671ab1ec
+checksums           rmd160  777b974353281b8e3b70549cabb77a6b1f0ee379 \
+                    sha256  2dee9ffdd16292878336a58d04a20f0ffe95555465fee7c9bd23b3490ef2abf3 \
+                    size    28844
 
-python.versions     27
+python.versions     27 36 37 38
 
 if {${name} ne ${subport}} {
     depends_build-append port:py${python.version}-setuptools
+    livecheck.type      none
+} else {
+    livecheck.name  netifaces
```

Comment:
this `else` block shouldn't be necessary; the default `pypi` livecheck set by the `python` PG should be sufficient

Reply:
Done, thanks for the review.